### ISSUE TITLE
Inline images within headers

### ISSUE THREAD

**Issue by D-Maxwell** [notable/notable#1656]
### Feature description

So using the standardized (`![AltText](path/to/file.png)`) syntax, it is possible to load and preview images, as well as doing so within a line, as to make the image loaded surrounded by text.
Except, this is not possible in conjunction with headers (`#`).

### Feature motivation

As I thought of decorating my headers with an icon, I realised this was in fact not possible : you either get a header, or an image, but they cannot be on the same line. I think it would be neat if we were able to do this; perhaps through a custom emoji if this change were to require too many modifications?

**Comment by MMK21Hub** [comment]
In the meantime, you can use HTML to add the icons, like this:
```html
<h1>
<img src="https://notable.app/favicon.ico" style="display:inline" width="32px" /> Bug testing
</h1>
```
![image](https://user-images.githubusercontent.com/50421330/211143615-b8cc6ab1-e953-4a26-b807-441811a6201d.png)

**Comment by D-Maxwell** [comment]
Yeah, I could, though I would've liked it if this was possible using pure markdown syntax. I mean it could surely be slightly rewired to allow for this right? If it can be done through HTML, there's no reason markdown couldn't do it.

**Comment by MMK21Hub** [comment]
Yeah, it should be possible - after all, it works in GitHub.

**Comment by D-Maxwell** [comment]
While this works, it needs manual refining to get the image size and position right ; and most importantly, declaring a header using an html tag (regardless of implementing an inline icon) appears to prevent it being listed in the table of contents, as well as the creation of a `@header` link.

Hopefully this comes to be implemented someday.

**Comment by D-Maxwell** [comment]
For now, this is the current setup I've got :
```md
<img src="C:\Users\maxwell\.notable\attachments\sliders-cubic-dark.png" style="position: absolute; transform: translate(25%,58%);" width="24px" />

## [ ‎ ‎ ] Section Title
```
This is about as clean as I got it to be (not all that much), and it is absolutely not dynamic.
It requires a whole three lines, manual tweaking, absolute paths, and empty spaces.
